docs/guide/bundles.md

```diff
@@ -24,59 +24,89 @@ To learn more about organizing your eZ Platform project, see [Best Practices](be
+|[ez-support-tools](https://github.com/ezsystems/ez-support-tools)|provides functionality for system information|
+|[ezplatform-design-engine](https://github.com/ezsystems/ezplatform-design-engine)|design fallback system for eZ Platform similar to legacy design fallback system|
+|[BehatBundle](https://github.com/ezsystems/BehatBundle)|common reusable sentence implementations and other common needs for Behat testing in eZ bundles/projects|
+
+!!! Enterprise
+
+    |Bundle|Description|
+    |---------|-----------|
+    |date-based-publisher|provides the date based publishing functionality for the eZ Studio product|
+    |flex-workflow|implementation of a collaboration feature that lets you send content draft to any user for a review or rewriting|
+
+### Optional bundles
+
+|Bundle|Description|
+|---------|-----------|
+|[ezplatform-i18n](https://github.com/ezsystems/ezplatform-i18n)|centralized internationalization|
+|[ezplatform-demo-assets](https://github.com/ezsystems/ezplatform-demo-assets)|contains binary install data for ezsystems/ezplatform-demo|
+|[PlatformUIBundle](https://github.com/ezsystems/PlatformUIBundle)|provides the main editing and back-end interface for eZ Platform v1|
+|[PlatformUIAssetsBundle](https://github.com/ezsystems/PlatformUIAssetsBundle)|assets for PlatformUIBundle, branches will only contain meta files and config, tags will contain the assets for eZ Platform v1|
+|[ezplatform-automated-translation](https://github.com/ezsystems/ezplatform-automated-translation)|adds automated translation|
+|[EzSystemsPrivacyCookieBundle](https://github.com/ezsystems/EzSystemsPrivacyCookieBundle)|adds privacy cookie banner|
+|[ezplatform-richtext](https://github.com/ezsystems/ezplatform-richtext)|Field Type for supporting rich formatted text stored in a structured XML format|
+|[CommentsBundle](https://github.com/ezsystems/CommentsBundle)|comment bundle for eZ Platform integrating with Disqus and Facebook and allowing custom integrations|
+|[EzSystemsRecommendationBundle](https://github.com/ezsystems/EzSystemsRecommendationBundle)|integration of YooChoose, a content recommendation solution|
+|[EzSystemsShareButtonsBundle](https://github.com/ezsystems/EzSystemsShareButtonsBundle)|adds social share buttons|
+|[RepositoryProfilerBundle](https://github.com/ezsystems/RepositoryProfilerBundle)|profiles Platform API/SPI and sets up scenarios to be able to continuously test to keep track of performance regressions of repository and underlying storage engines|
+|[QueryBuilderBundle](https://github.com/ezsystems/QueryBuilderBundle)|provides a PHP API dedicated to fluently writing repository queries|
+
+!!! Enterprise
+
+    |Bundle|Description|
+    |---------|-----------|   
+    |ezstudio-cron|eZ Platform EE 1.x series only - exposes cron/cron package for use in eZ Platform (or just plain Symfony) via a simple command `ezpublish:cron:run`|
+    |ez-service-network|provides API's and functionality for license handling and authentication needs|
```

Comment:
I don't think that this bundle is still in use in any version higher than old eZ Publish 5.4